datadog/tests/resource_datadog_synthetics_test_test.go

```diff
@@ -3051,6 +3067,51 @@ func createSyntheticsBrowserTestStepUserLocator(ctx context.Context, accProvider
 	}
 }
 
+const MMLCustomUserLocatorNoElement = `{"userLocator":{"failTestOnCannotLocate":true,"values":[{"type":"css","value":"user-locator-test"}]}}`
+
+func createSyntheticsBrowserTestStepUserLocatorNoElement(ctx context.Context, accProvider func() (*schema.Provider, error), t *testing.T, testName string) resource.TestStep {
+	return resource.TestStep{
+		Config: createSyntheticsBrowserTestUserLocatorNoElementConfig(testName),
+		Check: resource.ComposeTestCheckFunc(
+			testSyntheticsTestExists(accProvider),
+			resource.TestCheckResourceAttr(
```

Comment:
No need to test all those fields IMO, it's already tested somewhere else. You can only test what's specific to this test